CMakeLists.txt

```diff
@@ -232,21 +233,8 @@ endif(WIN32)
 STRING(TOUPPER ${CMAKE_BUILD_TYPE} GRCBTU)
 SET(COMPILER_INFO "")
 IF(MSVC)
-    IF(MSVC90)   #Visual Studio 9
-        SET(cmake_c_compiler_version "Microsoft Visual Studio 9.0")
-        SET(cmake_cxx_compiler_version "Microsoft Visual Studio 9.0")
-    ELSEIF(MSVC10) #Visual Studio 10
-        SET(cmake_c_compiler_version "Microsoft Visual Studio 10.0")
-        SET(cmake_cxx_compiler_version "Microsoft Visual Studio 10.0")
-    ELSEIF(MSVC11) #Visual Studio 11
-        SET(cmake_c_compiler_version "Microsoft Visual Studio 11.0")
-        SET(cmake_cxx_compiler_version "Microsoft Visual Studio 11.0")
-    ELSEIF(MSVC12) #Visual Studio 12
-        SET(cmake_c_compiler_version "Microsoft Visual Studio 12.0")
-        SET(cmake_cxx_compiler_version "Microsoft Visual Studio 12.0")
-    ELSEIF(MSVC14) #Visual Studio 14
-        SET(cmake_c_compiler_version "Microsoft Visual Studio 14.0")
-        SET(cmake_cxx_compiler_version "Microsoft Visual Studio 14.0")
+    IF(MSVC_VERSION LESS "${MSVC_MIN_VERSION}")
```

Comment:
I think "1800" doesn't really work with "VERSION_LESS", but it's the CMake/MSVC way of denoting versions; they don't do the major.minor or year.release scheme; citing /usr/share/cmake/Help/variable/MSVC_VERSION.rst:

> 
> MSVC_VERSION
> ------------
> 
> The version of Microsoft Visual C/C++ being used if any.
> If a compiler simulating Visual C++ is being used, this variable is set
> to the toolset version simulated as given by the ``_MSC_VER``
> preprocessor definition.
> 
> Known version numbers are::
> 
>   1200      = VS  6.0
>   1300      = VS  7.0
>   1310      = VS  7.1
>   1400      = VS  8.0 (v80 toolset)
>   1500      = VS  9.0 (v90 toolset)
>   1600      = VS 10.0 (v100 toolset)
>   1700      = VS 11.0 (v110 toolset)
>   1800      = VS 12.0 (v120 toolset)
>   1900      = VS 14.0 (v140 toolset)
>   1910-1919 = VS 15.0 (v141 toolset)
> 
> See also the  :variable:`CMAKE_<LANG>_COMPILER_VERSION` and
> :variable:`MSVC_TOOLSET_VERSION` variable.
>

Reply:
@michaelld does that make sense to you? If so, please consider merging :)

Reply:
I think `VERSION_LESS` will work with whatever you give it; doesn't to be "X.Y" etc ... could be just "X".

That said, `LESS` should also work since we're just talking about numbers.

My belief here is that since we're working with versions, we should use the `VERSION`version of `LESS`; more or less LOL ;)

Reply:
-THAT- said, I don't really care which we use, and we can always change this later if someone desires. Thus, I'm going to merge this PR since it looks like it should work & simplifies the code.